test_suite/dmqmc/np2/h2_simple_ks_complex/c_dmqmc.in

```diff
@@ -0,0 +1,28 @@
+system = read_in {
+	int_file = 'FCIDUMP',
```

Comment:
I wrote my own program to compute Hamiltonian elements between VASP's KS orbitals (VASP's integral dumping is not available for non-developers). I'll publish that program onto GitHub if it is necessary. In that case, might adding input for my own program and the INCAR file for VASP calculations be sufficient?

Reply:
There's also (@ajwt and @cjcscott know better than me) pyscf, which I think can do periodic systems.

Reply:
PySCF does have support for periodic systems, but its FCIDUMP module only produces real integrals (imaginary integrals for non-gamma-point calculations are ignored currently).
Though I believe with some simple modifications I can add support to it, the current Hamiltonian yields consistent results between complex DMQMC and the (unmodified) FCIQMC and FCI.

Reply:
So we have a private branch I believe that extracts complex integrals from pyscf. I will check with the team @ajwt @cjcscott

Reply:
(you can always to gamma point calculations for now where things are real)

Reply:
Yes, PySCF uses localized AO basis sets so gamma-point Hamiltonian of both HF and KS orbitals are real.

Reply:
This independent of the cell periodic part of the Bloch function (u_{nk}(r)). I mean, your plane wave integrals should be real too if evaluated at the gamma point, or at least could be made real.

Reply:
Oh, that's right. I used the standard k-point version of VASP where some phase shifts might exist so that the wave functions would become complex. The gamma-only VASP should be able to produce a purely real wave function in this case.